site/src/theme.ts

```diff
@@ -30,12 +30,25 @@ export const theme = {
             900: "#212121",
         },
     },
+    fontFamily: "Arial",
```

Comment:
Please add a fallback font

```suggestion
    fontFamily: "Arial, sans-serif",
```